components/_global-navigation.scss

```diff
@@ -57,10 +59,52 @@ $nav-width: 1024px;
 		}
 	}
 
+	&__icon-user {
+		border: solid 2px $color-dark-blue-gray;
+		border-radius: 50%;
+		fill: $color-dark-blue-gray;
+		height: 28px;
+		margin-right: 2px;
+		width: 28px;
+	}
+
+	&__notifications {
+		position: relative;
+	}
+
+	&__notification-counter-wrapper {
+		background-color: $color-tv;
+		border: 1px solid $color-white;
+		border-radius: $notification-counter-height / 2;
+		color: $color-white;
+		font-size: $typescale-size-minus-3;
+		height: $notification-counter-height - 2 * $notification-counter-border-width;
+		left: 6px;
+		line-height: $notification-counter-height - 2 * $notification-counter-border-width;
+		min-width: $notification-counter-height - 2 * $notification-counter-border-width;
+		position: absolute;
+		top: 7px;
+	}
+
+	&__notification-counter {
+		padding: 0 4px;
+	}
+
+	&__icon-bell {
```

Comment:
I don't think we should make bell as a separate component at this point. I don't see any additional use-case currently, so I'd suggest keeping it as it is. We can always reiterate when we find it required.